docs/framework/wcf/feature-details/limiting-message-distribution.md

```diff
@@ -13,15 +13,15 @@ The concept of `PeerHopCount` is similar to TTL (Time-To-Live) used in IP protoc
 
 Hop count may be added to a message by adding `PeerHopCount` as an attribute to the applicable property or field in the implementation of the message class. You can set this to a specific value before sending the message to the mesh. In this manner, you can use hop count to limit distribution of messages throughout the mesh when necessary, potentially avoiding unnecessary message duplication. This is useful in cases where the mesh contains a high amount of redundant data, or for sending a message to immediate neighbors, or neighbors within a few hops.
 
-- For code snippets and related information, see the [Peer Channel blog](https://go.microsoft.com/fwlink/?LinkID=114531).
+- For code snippets and related information, see the [Peer Channel blog](https://docs.microsoft.com/archive/blogs/).
```

Comment:
Found a cool site to see some redirections I can't check internally where they were going before 
https://wheregoes.com/

![image](https://user-images.githubusercontent.com/12971179/72588085-01908400-38ac-11ea-8dac-ff269a6e996e.png)

```suggestion
- For code snippets and related information, see the [The PeerHopCount Attribute: Controlling Message Distribution](https://docs.microsoft.com/archive/blogs/peerchan/the-peerhopcount-attribute-controlling-message-distribution) post on the Peer Channel blog.
```